.web-docs/components/builder/vsphere-supervisor/README.md

```diff
@@ -144,7 +144,7 @@ items are listed below as well as the _optional_ configurations.
 - `import_target_location_name` (string) - Name of a writable and import-allowed ContentLibrary resource in the namespace where the image will be imported.
 
 - `import_target_image_type` (string) - The type of imported image.
-  Defaults to `ovf`. Available options include `ovf`.
+  Defaults to suffix of the source URL. Available options include `ovf` and `iso`.
```

Comment:
The suffix may not always match the exact name of the type, a source URL could point to an ova file with `ova` suffix, but the the image type will be still `ovf`.

Reply:
Updated default to set to ovf from .ova suffix.